src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -19,73 +19,55 @@
     CopyColumnsTransform.UserName, "CopyColumns", "CopyColumnsTransform", CopyColumnsTransform.ShortName,
     DocName = "transform/CopyColumnsTransformer.md")]
 
-[assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(IDataView), typeof(CopyColumnsTransform), null, typeof(SignatureLoadDataTransform),
+[assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(IDataTransform), typeof(CopyColumnsTransform), null, typeof(SignatureLoadDataTransform),
     CopyColumnsTransform.UserName, CopyColumnsTransform.LoaderSignature)]
 
 [assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(CopyColumnsTransform), null, typeof(SignatureLoadModel),
     CopyColumnsTransform.UserName, CopyColumnsTransform.LoaderSignature)]
 
-[assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(CopyColumnsRowMapper), null, typeof(SignatureLoadRowMapper),
-    CopyColumnsTransform.UserName, CopyColumnsRowMapper.LoaderSignature)]
+[assembly: LoadableClass(typeof(IRowMapper), typeof(CopyColumnsTransform), null, typeof(SignatureLoadRowMapper),
+    CopyColumnsTransform.UserName, CopyColumnsTransform.LoaderSignature)]
 
 namespace Microsoft.ML.Runtime.Data
```

Comment:
>Runtime.Data [](start = 23, length = 12)

can you change it to Microsoft.ML.Transforms?

Reply:
The namespace also reflects the folder location. If we change the namespace, should this also move into Microsoft.ML.Transforms?

---
In reply to: [222523600](https://github.com/dotnet/machinelearning/pull/1141#discussion_r222523600) [](ancestors = 222523600)

Reply:
Actually I see some Transforms in Microsoft.ML.Transform that have the namespace Microsoft.ML.Runtime.Data - we should fix this.

---
In reply to: [222767195](https://github.com/dotnet/machinelearning/pull/1141#discussion_r222767195) [](ancestors = 222767195,222523600)